src/main/java/mate/academy/model/Driver.java

```diff
@@ -6,12 +6,30 @@
     private Long id;
     private String name;
     private String licenseNumber;
+    private String login;
+    private String password;
```

Comment:
let's add these fields to the equals and hashcode

Reply:
added.